src/exampleTypes/DateRangeFacet.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { contexturify } from '../utils/hoc'
+import { toNumber } from '../utils/format'
+import { FacetCheckboxList, displayFn, displayBlankFn } from '../utils/facet'
+
+let DateRangeFacet = ({
+  tree,
+  node,
+  hide = {
+    counts: false, // Hide the facet counts so only the labels are displayed
+  },
+  display = displayFn,
+  displayBlank = displayBlankFn,
+  formatCount = toNumber,
+}) => (
+  <div className="contexture-facet">
```

Comment:
we should change this to `contexture-daterangefacet`

Reply:
done.